### PR TITLE
Standardize and improve codeblocks

### PR DESCRIPTION
**Pull request by mirons-google**

### CODE REVIEW

File: _docs/howto/gsuite-group-collection.md [hunk 1]
```diff
@@ -55,13 +55,13 @@ After you
...
-gcloud compute copy-files PATH_TO_DOWNLOAD_KEY \
+```bash
+$ gcloud compute copy-files PATH_TO_DOWNLOAD_KEY \
     YOUR_USER@YOUR_INSTANCE_NAME:/tmp/service-account-key.json
 
-gcloud compute ssh YOUR_USER@YOUR_INSTANCE_NAME
+$ gcloud compute ssh YOUR_USER@YOUR_INSTANCE_NAME
 
-YOUR_INSTANCE: sudo mv /tmp/service-account-key.json THE_PATH_YOU_SPECIFIED_IN_DEPLOY_FORSETI.yaml
+$ YOUR_INSTANCE_NAME>: sudo mv /tmp/service-account-key.json THE_PATH_YOU_SPECIFIED_IN_DEPLOY_FORSETI.yaml
```
Comment (carise) [inline _docs/howto/gsuite-group-collection.md]: $ <YOUR_INSTANCE_NAME>

Comment (carise) [inline _docs/howto/gsuite-group-collection.md]: Actually never mind

File: _docs/howto/local-deployment.md [hunk 2]
```diff
@@ -55,53 +55,52 @@ To set up Cloud SQL for Forseti, follow the steps below:
...
 use the following command:
 
-```
-python build_protos.py
-python setup.py install
+```bash
+$ python setup.py install
 ```
 
 ### Executing Foreseti commands
```
Comment (carise) [inline _docs/howto/local-deployment.md]: Forseti

**Comment by blueandgold** [comment]
Adding `cla: yes` label, as the PR author is a googler at the time of this PR was created.

**Pull request merged**